students/dominic_martin/lesson03/list_lab.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env Python 3
+
+fruit = ['Apples', 'Pears', 'Oranges', 'Peaches', 'Strawberries',
+		'Bananas', 'Tangerines', 'Kiwis', 'Grapes']
+print(fruit)
+
+choice = int(input("What would you like to do? \nAdd Fruit - 1\n"
+		"Choose Fruit - 2\nDelete Fruit - 3\nLike Fruit - 4\n"
+		"Reverse Fruit - 5\nEnter Response:"))
+
+
+def addFruit(choice):
```

Comment:
These function names should be snake_case as well.  Also, make sure you're using 4 spaces for indentation.